docs/Getting_Started.md

```diff
@@ -0,0 +1,394 @@
+- **[Glitch](https://glitch.com/)**, an online development environment to build and host apps. You could also develop locally with a tool like **[ngrok](https://ngrok.com/)**.
+
+And here's what the finished app will look like:
+
+![Demo of example app](getting-started-demo.gif)
+
+To make the user flow a bit more explicit:
+1. User 1 initiates a new game and picks their object using the app’s `/challenge` slash command
+2. A message is sent to channel with an invitation to accept the challenge
+3. User 2 accepts the challenge and selects their object
+4. The result of the match is posted back into channel for all to see
+
+## Creating an app
+
+The first thing we’ll need to do is create an app. Navigate to [the developer dashboard](https://discord.com/developers/applications), then click **New Application** in the upper right corner.
+
+![App creation modal](app-create-modal.png)
+
+Enter a name for your app, then click **Create**.
+
+Once you create an app, you'll land on the **General Overview** page of the app's settings. Here you can view and configure basic information about the app, like its description and icon. You’ll also see an **Application ID** and **Interactions Endpoint URL**, which we’ll use a bit later in the guide.
+
+### Configuring a bot
+
+Next we'll add a bot user to your app, which allows it to appear in Discord similar to other members. On the left hand sidebar click **Bot**, then the **Add Bot** button.
+
+Once you create a bot, you’ll have an option to update its icon and username. Under that, there’s a **Token** section with a **Reset Token** button.
+
+![Bot tab in app setings](app-add-bot.png)
+
+Bot tokens are used to authorize API requests and carry all of your bot user’s permissions, making them *highly sensitive*. Make sure to never share your token or check it into any kind of version control.
+
+Go ahead and click **Reset Token**, and save the token somewhere safe.
+
+> warn
+> You won’t be able to view your token again unless you regenerate it, so make sure to keep it somewhere safe, like in a password manager. 
+
+### Adding permissions
+
+Apps need approval to perform actions inside of Discord (like installing a slash command or adding emojis). So before installing your app, let's add some permissions. Click on **OAuth2** in the left sidebar, then **URL generator**.
+
+> info
+> The URL generator is helpful to create an authorization link with the permissions your app needs to function. You can use the link to install your app onto your server, or share with others so they can install it on their own.
+
+For now, we’ll just add a couple of permissions:
+- `applications.commands` lets your app create commands in guilds its installed
+- `bot` is to enable your bot user. After you click bot, you can also add individual permissions. For now, check **Send Messages** and **Use Slash Commands**.
```

Comment:
Bots don't need the Use Application Commands permission, it only affects users. Since this is a guide aimed at new devs, it's probably better not to request that permission as bots typically don't need it (unless they happen to need to set permission overrides but this isn't the case here)

Reply:
Send Messages isn't needed too. A bot can reply to a slash command without `Send Messages` permissions.